### DISCUSSION TITLE
SCTP multihoming support in 0.9 release and release date

### DISCUSSION THREAD

**Discussion by parthibanethiraj** [loxilb-io/loxilb#244] [Q&A] [answered]
Hello, 
Can i know the  release date for 0.9 release of LoxiLB which will have SCTP multihoming support.

Also, will the support be there for kube-loxilb/k8s LB service level?

Regards

**Comment by nik-netlox** [comment, marked as answer]
Hi @parthibanethiraj,

We are currently working on release 0.9. We are receiving many requests from the community so we have postponed our current release. SCTP multihoming and many other features are expected to be released in June 2023 release. If you have any urgent requirements or requests then do contact us through our [slack](https://www.loxilb.io/members) channel. We will be happy to help.
